portal/get_patient_info.php

```diff
@@ -16,164 +16,182 @@
-    unset($_SESSION['password_update']);
-    $plain_code= $_POST['pass'];
-
-    $authorizedPortal=false; //flag
-    DEFINE("TBL_PAT_ACC_ON", "patient_access_onsite");
-    DEFINE("COL_PID", "pid");
-    DEFINE("COL_POR_PWD", "portal_pwd");
-    DEFINE("COL_POR_USER", "portal_username");
-    DEFINE("COL_POR_SALT", "portal_salt");
-    DEFINE("COL_POR_PWD_STAT", "portal_pwd_status");
-    $sql= "SELECT ".implode(",", array(COL_ID,COL_PID,COL_POR_PWD,COL_POR_SALT,COL_POR_PWD_STAT))
-          ." FROM ".TBL_PAT_ACC_ON
-          ." WHERE ".COL_POR_USER."=?";
-            $auth = privQuery($sql, array($_POST['uname']));
-if ($auth===false) {
-    $logit->portalLog('login attempt', '', ($_POST['uname'].':invalid username'), '', '0');
+// Settings that will override globals.php
+$ignoreAuth = 1;
+//
+
+// Authentication
+require_once('../interface/globals.php');
+require_once("$srcdir/authentication/common_operations.php");
+require_once("$srcdir/user.inc");
+$password_update = isset($_SESSION['password_update']);
+unset($_SESSION['password_update']);
+$plain_code = $_POST['pass'];
+
+$authorizedPortal = false; // flag
+DEFINE("TBL_PAT_ACC_ON", "patient_access_onsite");
+DEFINE("COL_PID", "pid");
+DEFINE("COL_POR_PWD", "portal_pwd");
+DEFINE("COL_POR_USER", "portal_username");
+DEFINE("COL_POR_SALT", "portal_salt");
+DEFINE("COL_POR_PWD_STAT", "portal_pwd_status");
+$sql = "SELECT " . implode(",", array(
+    COL_ID,
+    COL_PID,
+    COL_POR_PWD,
+    COL_POR_SALT,
+    COL_POR_PWD_STAT
+)) . " FROM " . TBL_PAT_ACC_ON . " WHERE " . COL_POR_USER . "=?";
+$auth = privQuery($sql, array(
+    $_POST['uname']
+));
+if ($auth === false) {
+    $logit->portalLog('login attempt', '', ($_POST['uname'] . ':invalid username'), '', '0');
     session_destroy();
-    header('Location: '.$landingpage.'&w');
-    exit;
+    header('Location: ' . $landingpage . '&w');
+    exit();
 }
 
 if (empty($auth[COL_POR_SALT])) {
-    if (SHA1($plain_code)!=$auth[COL_POR_PWD]) {
-        $logit->portalLog('login attempt', '', ($_POST['uname'].':pass not salted'), '', '0');
+    if (SHA1($plain_code) != $auth[COL_POR_PWD]) {
+        $logit->portalLog('login attempt', '', ($_POST['uname'] . ':pass not salted'), '', '0');
         session_destroy();
-        header('Location: '.$landingpage.'&w');
-        exit;
+        header('Location: ' . $landingpage . '&w');
+        exit();
     }
 
-    $new_salt=oemr_password_salt();
-    $new_hash=oemr_password_hash($plain_code, $new_salt);
-    $sqlUpdatePwd= " UPDATE " . TBL_PAT_ACC_ON
-      ." SET " .COL_POR_PWD."=?, "
-      . COL_POR_SALT . "=? "
-      ." WHERE ".COL_ID."=?";
-    privStatement($sqlUpdatePwd, array($new_hash,$new_salt,$auth[COL_ID]));
+    $new_salt = oemr_password_salt();
+    $new_hash = oemr_password_hash($plain_code, $new_salt);
+    $sqlUpdatePwd = " UPDATE " . TBL_PAT_ACC_ON . " SET " . COL_POR_PWD . "=?, " . COL_POR_SALT . "=? " . " WHERE " . COL_ID . "=?";
```

Comment:
for the future, need to have your editor stop making the changes like is done the 1 line above

Reply:
Why, it's format is correct plus I can't find any two PSR beautifiers that format the same anyway and this is getting to be a pain.

Reply:
@sjpadgett ,
Disregard my comment. It was late and I didn't realize those were variables (ie. I thought you were just concatenating a bunch of strings). Keep moving along, nothing to see here :)